src/views/Home/components/LostAndFoundCard/index.tsx

```diff
@@ -203,6 +229,21 @@ const LostAndFoundCard = () => {
             ) : null}
           </>
         )}
+        {foundReports && foundReports.length > 0 && (
+          <>
+            <Card>
+              <CardHeader title="Found" />
+            </Card>
+            {reportHeader('Found')}
```

Comment:
Only the items marked as found would show either way, since with the way it is, the items show as found, and when you mark it as picked up it changes the status to that, which means unless you mark an item as picked up, it would remain found in the system. That also solves the problem of only showing them when its new/clicking in it and having to deal with storing that information

Reply:
Yep, the functionality is the same, I'm only suggesting a text change of the displayed title to clarify exactly what you said, that this only shows the recently found items, the items that haven't been picked up yet.  So exactly what you said, but with wording shown to the user that is more specific.